sickbeard/subtitles.py

```diff
@@ -420,16 +420,25 @@ def merge_subtitles(existing_subtitles, new_subtitles):
     return current_subtitles
 
 
-def get_min_score():
+def get_min_score(video):
     """Return the min score to be used by subliminal.
 
     Perfect match = hash - resolution (subtitle for 720p is the same as for 1080p) - video_codec - audio_codec
+    Perfect match (no release_group): hash - release_group - audio_codec*
     Non-perfect match = series + year + season + episode
 
+    *Subliminal enhances video with metadata refiner and for some release names, guessit guess
+    audio_codec as DolbyDigital but subliminal changes it to AC3 -> https://github.com/Diaoul/subliminal/issues/677
+
+    :param video:
+    :type video: subliminal.video.Video
     :return: min score to be used to download subtitles
     :rtype: int
     """
     if sickbeard.SUBTITLES_PERFECT_MATCH:
+        if not video.release_group:
```

Comment:
Better to move this discussion outside this PR. They are related but this PR is done and shoes another problem... and the other discussion is just starting